easy_my_coop_taxshelter_report/models/tax_shelter_declaration.py

```diff
@@ -62,11 +62,11 @@ class TaxShelterDeclaration(models.Model):
 
     def _excluded_from_declaration(self, entry):

```

Comment:
Could you add a comment documenting that entry is a "subscription.register" object? I know it's not your code but it'd be more readable. :-)